apps/frontend/src/app/PageCharacter/CharacterDisplay/Tabs/TabTheorycraft/BuildConstaintCard.tsx

```diff
@@ -0,0 +1,143 @@
+import { StatIcon } from '@genshin-optimizer/gi-svgicons'
+import {
+  CardThemed,
+  CustomNumberInput,
+  CustomNumberInputButtonGroupWrapper,
+  DropdownButton,
+} from '@genshin-optimizer/ui-common'
+import { unit } from '@genshin-optimizer/util'
+import ClearIcon from '@mui/icons-material/Clear'
+import {
+  Box,
+  Button,
+  ButtonGroup,
+  CardHeader,
+  Divider,
+  ListItemIcon,
+  ListItemText,
+  MenuItem,
+  Stack,
+} from '@mui/material'
+import { useContext } from 'react'
+import { ArtifactStatWithUnit } from '../../../../Components/Artifact/ArtifactStatKeyDisplay'
+import type { MinTotalStatKey } from '../../../../Database/DataManagers/CharacterTCData'
+import { minTotalStatKeys } from '../../../../Database/DataManagers/CharacterTCData'
+import { CharTCContext } from './CharTCContext'
+import { useTranslation } from 'react-i18next'
+
+export function BuildConstaintCard({ disabled }: { disabled: boolean }) {
+  const { t } = useTranslation('page_character')
+  const {
+    charTC: {
+      optimization: { minTotal },
+    },
+    setCharTC,
+  } = useContext(CharTCContext)
+
+  return (
+    <CardThemed bgt="light">
+      <CardHeader title={t`tabTheorycraft.constraint.title`} />
```

Comment:
nit: This component doesn't match our other components from the optimize tab
![image](https://github.com/frzyc/genshin-optimizer/assets/36019388/5f8d4dfb-d5ad-44f5-92df-056ac28c5467)
![image](https://github.com/frzyc/genshin-optimizer/assets/36019388/1489a06a-03f7-474e-aff2-512fd029f4c4)

* Header font is too large
* Header isn't bolded
* Input doesn't go all the way to the edge (I don't really like that in the Optimize tab version, so I'm fine if that doesn't change)
* We could theoretically reuse the translations from the Optimize component